Back-end/schema/relations.js

```diff
@@ -0,0 +1,295 @@
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const { UserModel, DriverModel, OwnerModel, UserDTC, OwnerTC, DriverTC } = require('./models/User');
+const { ParkingTC } = require('./models/Parking');
+const { RentTC } = require('./models/Rent');
+const { ConversationTC } = require('./models/Conversation');
+const { MessageTC } = require('./models/Message');
+
+// User Relations
```

Comment:
let's make the comments separation stand-out more by using a docblock instead of a single line comment
```
/**
 * User Relations
 */
```